api/apps/articles/test/integration.coffee

```diff
@@ -63,6 +64,7 @@ describe 'articles endpoints', ->
         .send(title: 'Hi', partner_channel_id: ObjectId('5086df098523e60002000012')).end (err, res) ->
           res.body.title.should.equal 'Hi'
           done()
+      return
```

Comment:
[Mocha 3.0](https://github.com/mochajs/mocha/blob/master/CHANGELOG.md#300--2016-07-31) no longer lets you return a promise and inject a callback in a single test